### DISCUSSION TITLE
Appium 1.22.2 - safariTabBarPosition capability with simulator device - is there any impact of this capability on simulator?

### DISCUSSION THREAD

**Discussion by skumarigit** [appium/appium#16673] [Q&A]
I have tried safariTabBarPosition capability in my test and it has no impact, could you please suggest any way to see the imapct of this capability on simulator or in appium server logs. I have passed safariTabBarPosition capability as top but on navigating to the Safari window under settings it is showing the tab bar at the  bottom.

**Comment by svermagit** [comment]
Hi @mykola-mokhnach,

I have also tried to execute this capability with simulator and real devices both but it seems there is no impact with devices having iOS 15+  version, everytime the address bar in safari appears on bottom. Can you please advice something on this?

**Comment by KazuCocoa** [comment]
> Handle offset of Safari tab bar in nativeWebTap enabled interactions. 

So, this capability manages the offset. It is needed when you configure the tab bar position as https://github.com/appium/appium-xcuitest-driver/pull/1361 . This does not change the tab bar position. (The config change should be via UI interactions)

**Reply by KazuCocoa** [reply]
mm, `safariTabBarPositionOffset` might be better naming, now I think.
